fhir-audit/pom.xml

```diff
@@ -28,6 +28,10 @@
             <artifactId>fhir-core</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+        </dependency>
```

Comment:
We do not need this in audit.  AuditLogEntry is really a DTO and not the serialized object.

Reply:
Paul mentioned that he will make this change in a future PR.